setup.py

```diff
@@ -81,7 +81,7 @@
 setuptools_args = {}
 install_requires = setuptools_args['install_requires'] = [
     'ipython>=4.0.0',
-    'traitlets>=4.1.0',
+    'traitlets>=4.2.0',
```

Comment:
That is right, but traitlets 4.2 has (unrelated) bug fixes and is out there. I can revert that if you want.